### PR TITLE
Add prow setup for Nordix metal3-dev-tools and metal3-clusterapi-docs repo

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Comment by tuminoid** [comment]
Before we add prow config, should we be adding OWNERS to clusterapi-docs first and have you done other settings to clusterapi-docs you did for dev-tools earlier?

**Review by tuminoid — APPROVED** [review]
Besides my above comment, lgtm.

**Comment by kashifest** [comment]
> Before we add prow config, should we be adding OWNERS to clusterapi-docs first and have you done other settings to clusterapi-docs you did for dev-tools earlier?

Good point, other settings I have checked but OWNERS file should be added there first.

File: prow/config/config.yaml [hunk 1]
```diff
@@ -644,7 +679,7 @@ presubmits:
     - name: build
       branches:
         - main
-      run_if_changed: '^api|^test|^Makefile$'
+      run_if_changed: "^api|^test|^Makefile$"
```
Comment (furkatgofurov7) [inline prow/config/config.yaml:682]: i see single quotes being used consistently in the file though

**Review by furkatgofurov7 — APPROVED** [review]

File: prow/config/config.yaml (hunk 1 above)
Comment (tuminoid) [reply prow/config/config.yaml:682]: This is the same exact auto-formatting issue I had in my PR. I'm fairly sure that the double-quote is the de-facto formatting we should be using on all lines, but for whatever reason it only triggers the formatting on this one line. Anyways, revert to single-quote as @furkatgofurov7 suggested.

**Review by tuminoid — CHANGES_REQUESTED** [review]
Fix the single-quote issue.

File: prow/config/config.yaml (hunk 1 above)
Comment (kashifest) [reply prow/config/config.yaml]: Yeah it was the formatter. Updated it.

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Pull request merged**